dasharo-security/secure-boot.robot

```diff
@@ -233,6 +233,52 @@ SBO008.001 Attempt to enroll the key in the incorrect format (firmware)
     Select File In File Explorer    DB.txt
     Read From Terminal Until    ERROR: Unsupported file type!
 
+SBO009.001 Attempt to enroll the key in the incorrect format (OS)
+    [Documentation]    This test verifies that it is impossible to load
+    ...    a certificate in the wrong file format from the OS.
+    Skip If    not ${SECURE_BOOT_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_FIRMWARE_SUPPORT}    SBO008.001 not supported
+    Skip If    not ${TESTS_IN_UBUNTU_SUPPORT}    SBO008.001 not supported
+    Power On
+    ${sb_menu}=    Enter Secure Boot Menu And Return Construction
+    ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
+    Erase All Secure Boot Keys    ${advanced_menu}
+    Save Changes And Reset    3    5
+    Boot System Or From Connected Disk    ubuntu
+    Login To Linux
+    Switch To Root User
+    # remove existing directory that contains the keys
+    Execute Linux Command    rm -rf /usr/share/secureboot/keys
+    # make the directory structure
+    Execute Linux Command    mkdir /usr/share/secureboot/keys
+    Execute Linux Command    mkdir /usr/share/secureboot/keys/PK
```

Comment:
@WiktorG351 please use `mkdir -p`; this way we can omit creation of `../secureboot/keys` as it will be already created with PK dir, also, using -p is a good practice as it will not return error in cases where dir already exists

Reply:
@TomaszAIR done [here](https://github.com/Dasharo/open-source-firmware-validation/pull/150/commits/5811b042862c128816abe8ab4ab2a575d7eedb5b#diff-a85ad2551c590f23fcfdd0995aeb59134f1c5600bec9d6b563798c00053e51d4R254-R258)